tests/CommandsTest.php

```diff
@@ -18,7 +18,7 @@ public function setUp(): void
     {
         parent::setUp();
 
-        config(['tenancy.migrations_directory' => database_path('../migrations')]);
+        config('tenancy.migrations_paths', [database_path('../migrations')]);
```

Comment:
We need to use `config([])` because we are setting config values.

```suggestion
        config(['tenancy.migration_paths', [database_path('../migrations')]]);
```